.github/ISSUE_TEMPLATE/change.yml

```diff
@@ -0,0 +1,10 @@
+name: Change Request
+description: Suggest an idea.
+labels: ["enhancement"]
+body:
+  - type: textarea
+    id: change
+    attributes:
+      label: What should we change and why?
```

Comment:
I think you could have a few more options like screenshots or something here. Idk if this is enough info the capture everything that we are going to want always

Reply:
I intentionally left this very generic (basically identical to a standard existing issue with an automatically applied label for the project board) with the plan to create more customized forms as specific feature request use cases prove themselves out.

Reply:
But if you you have specific ideas I'm happy to add more.

Reply:
The only idea I have is to separate code, cli, and gui forms for example and require screenshots for the gui + doc, command for cli + doc, or github link for the code + doc but I don't think those are great ideas. I would prefer to just wait for conditional issue templating